handlers/guestbook.go

```diff
@@ -74,7 +74,7 @@ func registerGuestbookHandlers(app *pocketbase.PocketBase) {
 			return c.HTML(http.StatusOK, html)
 		})
 
-		e.Router.GET("/guestbook/addMessage", func(c echo.Context) error {
+		e.Router.GET("/addMessage", func(c echo.Context) error {
```

Comment:
Why did you remove the namespace for that endpoint?

Reply:
Because i am not proficient enough yet with htmx/pocketbase, and that setup resulted in mc-area being rendered in mc-area again. This avoids that problem.